Documentation/Troubleshooting/disaster-recovery.md

```diff
@@ -57,18 +57,9 @@ the CRs to their prior state without even necessarily suffering cluster downtime
     kubectl -n rook-ceph get configmap -o yaml > configmaps.yaml
     ```
 
-3.  Back up and remove the `ValidatingWebhookConfiguration`. This is the resource which connects Rook custom resources to the
-    operator pod's validating webhook. Because the operator is unavailable, we must temporarily disable the valdiating webhook in
-    order to make changes.
-
-    1.  Save the `ValidatingWebhookConfiguration` responsible for controlling Rook resources. (These are not namespaced
-        resources.)
-
-        ```console
-        kubectl get ValidatingWebhookConfiguration rook-ceph-webhook -o yaml > rook-ceph-webhook.yaml
-        ```
-
-    2.  Delete the `ValidatingWebhookConfiguration`.
+3.  Delete the `ValidatingWebhookConfiguration`. This is the resource which connects Rook custom resources
+    to the operator pod's validating webhook. Because the operator is unavailable, we must temporarily disable
+    the valdiating webhook in order to make changes.
```

Comment:
They may not have the admission controller enabled, so how about a comment that they can skip this step in that case?